### ISSUE TITLE
[NPC] Shattered Hand Executioner -loot- Shattered Halls

### ISSUE THREAD

**Issue by kDracku** [WoWManiaUK/Blackwing-Lair#1921]
**Links:**

https://www.wowhead.com/npc=17301/shattered-hand-executioner

**What is happening:**
The NPC drops Badge of Justice
http://tinypic.com/view.php?pic=2cfuu77&s=9#.XN1mjqRS-M8
**What should happen:**
Mostly they drop Primals or some random greens

**Comment by Echinaceax** [comment]
yes Badge of Justice currency needs removing, this currency is not in Cata @mfranca0009
